sdk/eventhub/azure-eventhub/azure/eventhub/_common.py

```diff
@@ -273,7 +273,8 @@ class EventDataBatch(object):
     at which point a `ValueError` will be raised.
     Use the `send_batch` method of :class:`EventHubProducerClient<azure.eventhub.EventHubProducerClient>`
     or the async :class:`EventHubProducerClient<azure.eventhub.aio.EventHubProducerClient>`
-    for sending. The `create_batch` method accepts partition_key as a parameter for sending a particular partition.
+    for sending. The `create_batch` method accepts partition_key or partition_id as a parameter
+    for sending to a particular partition.
```

Comment:
partition_key isn't a particular partition.

Reply:
I find it's better to just remove the comment about `create_batch`, user can read the comment of `create_batch` in `EventHubProducerClient`.